### DISCUSSION TITLE
A Flexible Deflationary Model for KTON – Empowering Holders, Boosting Staking Rewards, and Strengthening the DAO

### DISCUSSION THREAD

**Discussion by sonoferin2000** [ringdao-community-guild/discussion#193] [Ideas]
Objective:
Let’s introduce a gradual deflationary model for KTON that benefits everyone—from holders to the DAO itself. The goal is simple: create a steady revenue stream, reward those who stake, and build a treasury reserve that can be tapped for future liquidity. This proposal is all about flexibility and community choice, bringing a sustainable, deflationary approach that can evolve over time.

Key Proposal Elements:
Deflationary Transaction Fee (Up to 1%—Community Decides the Final Rate)
A small transaction fee would apply whenever KTON changes hands, excluding transfers between liquidity providers or smart contracts. Recommended Rate: I'm  suggesting a 1% max as a starting point, but the exact rate is completely open to community feedback to keep trading healthy.But feel it should be kept minimal so that it isn't seen so much as a burden,while still accomplishing what it is put in place to accomplish.

Distribution of the Fee:
Here’s how I'm thinking the fee could be split:
Burn Mechanism: Half of each fee would go toward a burn, gradually reducing the total KTON supply.
DAO Contribution: The remaining half could be:
Sent to the KTON DAO treasury, or
Split, with part going to the treasury and part to a rewards pool for stakers, or
Another distribution the community feels works best.
Note: All numbers here are just examples—I'm looking to fine-tune everything with your input. we can play with the numbers and ratios, and design something that strikes the right ballance.

Positive Impact on KTON Token Value:
Supply Reduction and Value Growth: As KTON gets burned with each transaction, the overall supply shrinks. Over time, this scarcity can drive up
the token’s value as demand grows or holds steady.
Incentives for Long-Term Holding: With a portion of fees going back to stakers, there’s a built-in reward for those who choose to hold and support the ecosystem.
Boosting Demand for Staking: Redistributing part of the fee to stakers creates ongoing rewards, drawing more holders to stake and contributing to a more stable token price.
Enhanced Liquidity and Stability: A gradually growing DAO treasury supports liquidity, making the whole ecosystem more resilient.

Community-Driven Adjustments:
All suggested numbers—fee rates, distributions—are flexible and open to community input. We want this fee to feel beneficial, not like a burden.

An Alternative to Buybacks:
This deflationary model is a resource-light approach compared to traditional buybacks. By building up the DAO treasury over time, the DAO can influence market stability and liquidity without needing to rely on big buybacks.

Conclusion:
In short, this proposal aims to set up a deflationary model that rewards KTON holders and grows the DAO’s self-sustaining revenue. By encouraging long-term staking and reducing the circulating supply, it lays the foundation for gradual value growth, liquidity, and community governance.

Community Discussion:
KTON holders are invited to share thoughts, suggest tweaks, and help shape this proposal. All the figures here are just examples, meant to kickstart the conversation. Let’s work together to align this model with our shared vision for the DAO’s future.

**Comment by sonoferin2000** [comment]
I hope the community can get behind this proposal and help to polish it up, so that it can have the most positive impacts with the least downsides. i feel it is important to not impact liquidity providers with this tax, as they are taking on a lot of risk to allow tokens to be tradable. so I feel they should be exempted.

**Comment by SasoLithops** [comment]
Hi!

Before getting to your idea I think one main thing should be put in spotlight, that being we must not fill the KTON DAO treasury with a lot of KTON as this could potentialy lead to KTON DAO being unable to use these funds as selling them would influence the price too much. I was part of one DAO that became not functional exactly because of this. They own only their token and selling it for development would send its price to zero as poor liquidity.

So let's try with example so I understand more.

I buy 100 KTON on DEX. 
I pay 1% which is 1 KTON fee.
That 1 KTON gets divided:
0,5 KTON gets burned
0,5 KTON gets to KTON DAO. (we will still discuss how much of that could be given to stakers)

I have no opposition to that. Also 1% is ok to me. That could imho definitely be **first way** of funding KTON DAO.

I am doing on a side research since we KTON DAO give KTON to loyal RING DAO RING stakers, we should continue to receive some RING from RING DAO in return. I plan to publish a Github debate on how we could use this Ring which could be a **second way** of funding KTON DAO. The more income DAO gets the better it will thrive. However we must be careful not to hurt the stakers/holders too much as atm they/we take all the heat in those changes, so rewards should be good.
I plan to implement a change that will require active participation in governence as we could receive tokens we chose in governance. But more on that next time.

We also must address liquidity providers incentives asap as they have now stopped.

Back to idea. I would vote AYE.

**Comment by sonoferin2000** [comment]
I wasn't thinking of dao selling KTON, more  had  liquidity provision in mind. if too much starts to accumulate in treasury,we have option to vote for changing the distribution ratio, or to lower the tax.

**Comment by sonoferin2000** [comment]
discussion has been transfered to https://github.com/darwinia-network/KtonDAO/discussions/25 .Please follow it there
